lib/vsc/administration/slurm/sync.py

```diff
@@ -257,11 +276,11 @@ def slurm_institute_accounts(slurm_account_info, clusters, host_institute):
         for (inst, vo) in INSTITUTE_VOS_BY_INSTITUTE[host_institute].items():
             if inst not in cluster_accounts:
                 commands.append(
-                    create_add_account_command(account=inst, parent=None, cluster=cluster, organisation=inst)
+                    create_add_account_command(account=inst, parent=None, cluster=cluster, organisation=inst, fairshare=500)
                 )
             if vo not in cluster_accounts:
                 commands.append(
-                    create_add_account_command(account=vo, parent=inst, cluster=cluster, organisation=inst)
+                    create_add_account_command(account=vo, parent=inst, cluster=cluster, organisation=inst, fairshare=500)
```

Comment:
This can be whatever, except for gvo00012, I guess.

Reply:
Well, @wpoely86 should probably have a say in it :p since they have a similar setup.

Reply:
handle it in a dedicated function or worstcase a class.

Reply:
Why not just take the value from the AP here too?

Reply:
@wpoely86 yes ofcourse. this needs a chaneg in the AP VO model and in the ap clients code.
i guess the default value in the AP is not so relevant?

Reply:
As far as I know all VO have a fixed value of '100' currently in the AP.

In any case, we don't have slurm in production yet so as long as I can change the values for brussel, I'm good.
But as far as I possible I would like to keep it identical to your case.

Reply:
ok, didn't know that. default is indeed 100 atm.
@itkovian up to you if you want this default changed or not in the AP.

Reply:
For the VOs, it can pretty much be whatever, but the institutes should have some mapping we get from a config file, since 8500 shares for UGent at the VUB versus 500 shares for the VUB at the VUB will likely not fly :)

Reply:
what do you mean? i'm missing something that you have probably explained me a few times ;)

Reply:
Yes, same question as Stijn ;)

We already have a VO per institute, do we also need to set shares at institute level too?

Reply:
Yes, since those are higher up in the tree.

Reply:
We only create accounts for the `gvo*` groups. Sorry, the tree example was wrong :) will fix.